.gitignore

```diff
@@ -36,3 +36,4 @@ node_modules
 .vendor
 Goopfile.lock
 venv/
+.coala.log
```

Comment:
people who use coala are developers :) (although not for coala ;D)
Your choice :)